styles/widgets/generic/gridBase.generic.less

```diff
@@ -865,6 +865,30 @@
         &.dx-scrollable-scrollbars-alwaysvisible.dx-scrollable-both .dx-scrollable-wrapper .dx-scrollable-container .dx-scrollable-content {
             padding-right: 0;
         }
+
+        td {
+            &.dx-validation-pending {
+                position: relative;
+                padding: 0;
+
+                .dx-highlight-outline {
+                    padding-right: @GENERIC_TEXTEDITOR_INVALID_BADGE_SIZE;
+
+                    .dx-rtl&,
+                    .dx-rtl & {
```

Comment:
Ask Babich about this style, because he is removing such constructions to support SSAS

Reply:
These constructions will be converted automatically. There should not be problems.